Yosemite/YosemiteTests/Stores/ProductReviewStoreTests.swift

```diff
@@ -123,9 +123,134 @@ final class ProductReviewStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    /// Verifies that `ProductReviewAction.synchronizeProductReviews` for the first page deletes stored Product Reviews for the given site ID.
+    ///
+    func test_syncing_product_reviews_on_the_first_page_resets_stored_product_reviews() {
+
+        // Given
+        let reviewID1: Int64 = 1
+        let reviewID2: Int64 = 2
+        let productReviews = [self.sampleProductReview(reviewID: reviewID1), self.sampleProductReview(reviewID: reviewID2)]
+        self.store.upsertStoredProductReviews(readOnlyProductReviews: productReviews,
+                                              in: self.viewStorage,
+                                              siteID: self.sampleSiteID)
```

Comment:
nit: `self` probably not needed?

```suggestion
        store.upsertStoredProductReviews(readOnlyProductReviews: productReviews,
                                          in: viewStorage,
                                          siteID: sampleSiteID)
```